lib/bullet.rb

```diff
@@ -26,7 +26,7 @@ module Bullet
   if defined?(Rails::Railtie)
     class BulletRailtie < Rails::Railtie
       initializer 'bullet.configure_rails_initialization' do |app|
-        app.middleware.use Bullet::Rack
+        app.middleware.insert_before ActionDispatch::ContentSecurityPolicy::Middleware, Bullet::Rack
```

Comment:
could it break rails 4 app?

Reply:
Good catch; I've added a guard to prevent that.